CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Kubernetes executor prints exception [#23](https://github.com/ncsa/datawolf/issues/23)
+- Clean dependency updates [#27](https://github.com/ncsa/datawolf/issues/27)
```

Comment:
Can you change this entry to something like "hsqldb to 2.7.3" so it's more descriptive of what was done? Thanks! After that, I think it's ready to merge.

Reply:
Sure, just did